packages/blockchain-wallet-v4-frontend/src/hooks/useCoinBalances/useCoinBalances.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+
+import { getBalanceSelector } from 'components/Balances/selectors'
+import { RootState } from 'data/rootReducer'
+
+import { CoinDataItem } from '../../scenes/extension/CoinView/types'
+
+export const useCoinBalances = () => {
+  const state = useSelector((state: RootState) => state)
+  const [coins, setCoins] = useState<CoinDataItem[] | null>(null)
+
+  const allowedChains = ['ETH', 'BTC', 'XLM', 'BCH', 'STX']
+
+  useEffect(() => {
+    const getCoins = () => {
+      const coinsArr: CoinDataItem[] = []
+
+      // TODO: Check active wallet
+      Object.entries(window.coins).forEach((coin: any) => {
```

Comment:
can you deconstruct the entry i.e `Object.entries(window.coins).forEach(([coin, coinfig])` and replace coin[0] and coin[1]

Reply:
Also `any` should be avoided wherever possible.